latch_cli/snakemake/single_task_snakemake.py

```diff
@@ -37,6 +38,11 @@ def eprint(x: str) -> None:
 rules = data["rules"]
 outputs = data["outputs"]
 
+non_blob_parameters = data.get("non_blob_parameters", {})
+
+sw = sys.modules["snakemake.workflow"]
```

Comment:
I'd much rather we monkey patch whatever is supposed to load the config?
e.g. `snakemake.workflow.Workflow.configfile`

Reply:
or call `snakemake.utils.update_config`?

Reply:
`snakemake.workflow.config` is not even defined in my installation of snakemake

Reply:
i added a todo